suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -797,9 +797,7 @@ export const submitPassphrase = createThunk(
         dispatch(
             discoveryActions.updateDiscovery(
                 {
-                    status: 'progress',
-                    progress: 0, // dummy value, otherwise it comes in progress event from trezor-connect
-                    total: 100,
+                    status: 'passphrase-submitted',
```

Comment:
this is the important change, I am not sure why it was there like this, but this I think works better. Hopefully it won't break anything.

Reply:
It's manual. But will check

Reply:
I changed it with a simple flag